### DISCUSSION TITLE
Turning off formulas completely

### DISCUSSION THREAD

**Discussion by bart84** [iddan/react-spreadsheet#362] [Q&A] [answered]
Hello. 

I want to use react-spreadsheet but without using formulas at all.

Is there a way to disable formulas ( formula parsing ) with one flag for example?

I know there is an example at the main web page to override the formulas ( for example SUM: undefined):
https://iddan.github.io/react-spreadsheet/docs/formula-parser

But it didn't work for me. I can still SUM with =SUM(2+2) or by pointing to cells like this =SUM(B1,B2). And I see the formulas are being parsed and I get the result, hence the formula parser still works.

I am using TypeScript in my project. 

Thanks in advance for an answer!

Best!

**Comment by iddan** [comment, marked as answer]
Hey @bart84 at the moment there is no such option but I think you can customise the cell viewer to always view the raw data.
If you'd like to create a PR I'd be happy to review it
